tools/github/setup.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#if [[ $TEST_SUITE =~ Dummy ]]
```

Comment:
This might need to be deleted?

Reply:
no, if you do not want to defeat the goal of the dummy test to avoid to run a fast build withou waiting an hour...

Reply:
I love this idea to skip tests for debugging.
How about making this as an input as well just like debugging enablement option so that users can easily trigger this feature without changing any code?

Reply:
it is already there...if you launvh the dummy build and enable ngrok in a few second you get an ssh command to get into the vm with everythingalready setup to build and test.

Reply:
Can we control whether to run these dummy tests like this?
![image](https://user-images.githubusercontent.com/3447251/208024370-1d2a9faa-724c-4225-a0cd-5b5153d0708d.png)

Also, there is the same code block, so can we delete these comments?
https://github.com/apache/openwhisk/pull/5361/files#diff-fc5257f5200a46b6baab3ca657b694cc84dc77e3a5f9d71062b25fce994c02b4R70

Reply:
Yes you control the test from there. Select "Dummy" to run the dummy test. It is useful when also you enable Ngrok debugging so you get an ssh command to get into the CI build VM to experiment.

Reply:
ah ok I got confused.